components/SlideOver.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Drawer from 'react-modern-drawer';
+import 'react-modern-drawer/dist/index.css';
+import ClientOnly from './ClientOnly';
+
+export default function SlideOver({
+  isOpen,
+  onClose,
+  children,
+}: {
+  isOpen: boolean;
+  onClose: (status: boolean) => void;
+  children: React.ReactNode;
+}) {
+  return (
+    <>
+      <ClientOnly>
```

Comment:
why does it need to be ClientOnly?

Also since the fragment (`<>`) has only one item, you can remove it and just return the inner item